diff --git a/dist/components/StateHeader.js b/dist/components/StateHeader.js
index fa72eac6837ec54b2ef8b1b0815c302edb5ecf8a..c964c4526de2bb4f3ec0ff901074e2e42ca4b596 100644
--- a/dist/components/StateHeader.js
+++ b/dist/components/StateHeader.js
@@ -2,7 +2,7 @@ import { useStyles } from "../styles/use-styles.js";
 import { createSignal, onCleanup, onMount } from "solid-js";
 import { className, effect, insert, template } from "solid-js/web";
 import dayjs from "dayjs";
-import relativeTime from "dayjs/plugin/relativeTime";
+import relativeTime from "dayjs/plugin/relativeTime.js";

 //#region src/components/StateHeader.tsx
 var _tmpl$ = /* @__PURE__ */ template(`<div><div></div><div style=display:flex;align-items:center;gap:16px><div><div>Key</div><div></div><div>Last Updated</div><div> (<!>)</div></div><div style=margin-left:auto;font-weight:bold>% reduction`);
