--- .venv/lib/python3.8/site-packages/selenium/webdriver/chrome/options.py	2022-05-12 17:07:24.000000000 -0400
+++ .venv/lib/python3.8/site-packages/selenium/webdriver/chrome/options.py.bak	2022-05-12 17:08:17.000000000 -0400
@@ -121,7 +121,7 @@
         to the ChromeDriver
 
         :Args:
-         - extension: path to the \*.crx file
+         - extension: path to the \\*.crx file
         """
         if extension:
             extension_to_add = os.path.abspath(os.path.expanduser(extension))
